DXMainClient/Domain/Multiplayer/Map.cs

```diff
@@ -367,6 +375,35 @@ public bool SetInfoFromMpMapsINI(IniFile iniFile)
             }
         }
 
+        private void GetTeamStartMappingPresets(IniSection section)
+        {
+            TeamStartMappingPresets = new List<TeamStartMappingPreset>();
+            for(var i = 0;;i++)
```

Comment:
```suggestion
            for (var i = 0; ; i++)
```

Reply:
Additionally, as defined in the style guide:

> Never use `var` with primitive types.